.github/workflows/ci.yaml

```diff
@@ -4,7 +4,7 @@ on: push
 # Ref: https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#permissions
 permissions: read-all
 env:
-  MAKEFLAGS: '-j 2'
+  MAKEFLAGS: "-j 2"
```

Comment:
I think we can revert the changes to this file (were you attempting to fix the failing tests related to Git submodules?).

Reply:
you're right, I had the wrong impression that I reverted all the changes I made to this file